flybywiresim/a32nx
# feat: incorporate new model changes

## Summary of Changes

In sim update 2, Asobo updated the interior model and I think the exterior, so I included both. This works with the wiper changes after renaming some nodes and splitting animations.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

* Check speed placard is not overlayed with another one from the default A320
* Check that BRK FAN button has a normal cursor and says `(Inop.)`


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
